edge-modules/TestAnalyzer/ResponseStatus.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer
+{
+    using System;
+
+    public class ResponseStatus
```

Comment:
Where is this class used? to pass test result from DirectMethod module and Twin update module to Analyzer?  Current name is too general, please be specific.

Reply:
Yes, that is where it is used. Good suggestion, I will update.